packages/wallet-sdk/src/CoinbaseWalletSDK.ts

```diff
@@ -51,7 +48,7 @@ export interface CoinbaseWalletSDKOptions {
 }
 
 export class CoinbaseWalletSDK {
-  public static VERSION = SDK_VERSION;
+  public static VERSION = LIB_VERSION;
```

Comment:
I think we should still import the package json as a fallback. `require('../package.json').version` This should happen outside of the class

Reply:
@cb-jake I could import it, but it's unreachable code / it will never get referenced/used.
(LIB_VERSION is hard coded string in version.ts file which gets updated for each release, it's truthy value)

Reply:
Aww, I was thinking the version file only existed when a build happens. Nvmd. This looks good as is. Thanks!

Reply:
All good :+1: